src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/csvjsonconvertor.py

```diff
@@ -71,29 +71,29 @@ def read_csv(csvFile):
     return csvdict
 
 # generate result object with raw values
-def generate_result_object(resultvalues):
+def generate_result_object(resultvalues, metric):
     result = serialobj.Result()
     # generate measurement result test object per iteration
     measurements = list()
     measurement = serialobj.Measurement()
     measurement.iterValues = resultvalues
-    measurement.measurementType = 'execution_time'
```

Comment:
nit: Not a fan of properties that start with their type name.  I'd prefer measurement.Type.